docs/changes.rst

```diff
@@ -2,6 +2,11 @@
 Change Log
 ==========
 
+9.1.0
```

Comment:
FYI, I'm going to be releasing nebuchadnezzar as `9.1.0b1` <-- Note the `b1`. This specifies this as a pre-release, thus pip won't install it on the Content Managers system without them explicitly requesting it.

This will enable us to complete the BakedPDF CLI parts of nebuchadnezzar without interrupting the Content Manager's existing installation. And it will also mean the cnx-automation tests won't fail while we are still in development.

Reply:
Oh I see!  That's handy. :+1: